atlp-rwanda/hackers-ec-Fe
# #187419000 Configure ESLint and Git Pre commit hook  And Hound CI

#### Configure ESLint and Git Pre commit hook to run tests and ESlint format before commits And Hound CI

#### How should this be manually tested?

 Clone the repository branch in the local Machine
 perform npm install to have all packages
 Change something in a  files and commit
  See how the configured tools works, like not using imported files and declaring variables and not use them

#### What are the relevant pivotal tracker/Trello stories?

#187419000
